polus-apply-flatfield-plugin/src/log4j.properties

```diff
@@ -0,0 +1,12 @@
+# Copied from:
```

Comment:
Is this file not included with the bfio utility image?

Reply:
It isn't. What I can do is set up bfio to handle the javabridge so that this all happens behind the scenes. I've been meaning to set it up to do this. For now, it isn't going too happen. I'll open a ticket. For now, the cookiecutter template generates this file and gives boilerplate for initializing log4j properly.